x-pack/plugins/secops/public/components/page/network/network_dns_table/index.tsx

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiBadge, EuiFlexGroup, EuiFlexItem, EuiIconTip } from '@elastic/eui';
+import { isEqual } from 'lodash/fp';
+import React from 'react';
+import { connect } from 'react-redux';
+import styled from 'styled-components';
+import { ActionCreator } from 'typescript-fsa';
+
+import {
+  Direction,
+  NetworkDnsEdges,
+  NetworkDnsFields,
+  NetworkDnsSortField,
+} from '../../../../graphql/types';
+import { networkActions, networkModel, networkSelectors, State } from '../../../../store';
+import { Criteria, ItemsPerRow, LoadMoreTable } from '../../../load_more_table';
+
+import { getNetworkDnsColumns } from './columns';
+import { IsPtrIncluded } from './is_ptr_included';
+import * as i18n from './translations';
+
+interface OwnProps {
+  data: NetworkDnsEdges[];
+  loading: boolean;
```

Comment:
Thoughts on pulling props like `loading`, `type`, etc up to their own aptly-named `interface`? Something I was thinking about as was making the `IP Overview` component and contemplating ways we can streamline component development + readability.